### PR TITLE
Standarize alt links

### PR DESCRIPTION
**Pull request by cvvergara**
List taken from http://www.osgeo.org

Standarized the alt & the link for the OSGeo projects and OSGeo community projects .
efforts not done on
- not clear to me what were they
- retired projects on V11

## OSGeo Projects

- [x]    GRASS GIS
- [x]    gvSIG
- [x]    GeoNode
- [x]    Marble
- [x]    QGIS
-  FDO  - no documentation
- [x]    GDAL/OGR
- [x]    GEOS
- [x]    OSSIM
- [x]    PostGIS
- [x]    GeoNetwork
- [x]    pycsw
- [x]    deegree
- [x]    geomajas
- [x]    GeoMOOSE
-  GeoServer -- all had the logo
- [x]    Mapbender
- [x]    MapFish
- [x]    MapGuide Open Source
- [x]    MapServer
- [x]    OpenLayers
- Orfeo ToolBox (OTB) -- wait until it is official
- pywps -retired
- GeoTools - retired

## OSGeo Community Projects

- [x] istsos
- [x] zoo-project
- [x] proj4
-  team engine - no documentation
-  Geoinformatica  - no documentation
-  GeoWebCache   - no documentation
- [x] MapProxy
- [x] pgRouting
- Postal Address Geo-Coder   - no documentation
-  Virtual Terrain Project  - no documentation

## marked as OSGeo but not OSGeo
(maybe subprojects) so I didnt do nothing

- qgismapserver
- tinyows

### CODE REVIEW

**Comment by cvvergara** [comment]
@jodygarnett 
Can you verify the changes please.
You can verify by commit, clicking on the commit number it will show only the files changed in the commit

**Comment by jodygarnett** [comment]
Thanks for the legwork @cvvergara, checking now.

File: en/overview/proj4_overview.rst [hunk 1]
```diff
@@ -3,10 +3,16 @@
 :Version: osgeo-live11.0
 :License: Creative Commons Attribution 3.0 Unported (CC BY 3.0)
 
-.. image:: ../../images/project_logos/logo-proj4.png
+.. image:: /images/project_logos/logo-proj4.png
   :alt: project logo
   :align: right
```
Comment (jodygarnett) [inline en/overview/proj4_overview.rst]: I am not sure if PROJ is an osgeo project or not.

* https://github.com/OSGeo/proj.4
* http://proj4.org

So yes, it is part of MetaCRS so it is a community project.

Comment (jodygarnett) [inline en/overview/proj4_overview.rst]: So this one should have a community project blurb.

**Comment by cvvergara** [comment]
@jodygarnett thanks for reviewing,
Some proj 4 files already had the community Logo. I just fixed the link to their logo also.

Here is an example that shows the blurb:
https://github.com/cvvergara/OSGeoLive-doc/blob/593d58336cbf6b8a557efe7fe53cdd1cec8acd68/en/overview/proj4_overview.rst

blurb & legwork: I learned 2 words today

**Pull request merged**